src/mqtt_client/AtlasMqttClient.cpp

```diff
@@ -0,0 +1,78 @@
+#include "AtlasMqttClient.h"
+#include "AtlasMqttClient_callback.h"
+#include "AtlasMqttClient_actionListener.h"
+#include "AtlasMqttLogger.h"
+#include "AtlasMqttException.h"
+#include <mqtt/async_client.h>
+
+namespace Atlas
+{
+AtlasMqttClient* AtlasMqttClient::instance = nullptr;
+
+AtlasMqttClient& AtlasMqttClient::getInstance()
+{
+    if (instance == nullptr)
```

Comment:
Please use a construction like this here:
static AtlasMqttClient instance;

return instance;

Reply:
Done